metricbeat-tests/metricbeat.go

```diff
@@ -47,10 +50,16 @@ func RunMetricbeatService(version string, monitoredService services.Service) (se
 	service.SetBindMounts(bindMounts)
 	service.SetCmd(strings.Join(setupCommands, " "))
 	service.SetEnv(env)
+	service.SetAsDaemon(true)
 	service.SetLabels(labels)
 
-	container, err := service.Run()
-	if err != nil || container == nil {
+	fn := func(ctx context.Context) error {
+		return deleteIndex(ctx, "metricbeat", indexName)
+	}
+	service.SetCleanUp(fn)
```

Comment:
I think that it would be even more important to delete the indexes before running the scenario to ensure that they run in a clean environment.